### PR TITLE
Update Proposal UI to new Mock up

### LINKED ISSUE #598: add politeia welcome page
Currently on the proposals page, proposals are being downloaded automatically.

What is expected is that when the cx accesses the Proposals page for the first time, they should see a welcome screen just like the StakeShuffle page, with a button at the bottom asking if they will like to download proposals. 

Only then are the proposals to be downloaded based on the users consent.

image

### LINKED ISSUE #600: Proposals page nav width
When the proposals page is displayed in full screen, the proposal navigation tabs are expanded and there is currently too much whitespace.
image

The navigation tabs should retain their default width even in full screen. If need be, the tabs can be left aligned on full screen.

### LINKED ISSUE #683: proposals: Crash on tab navigation.
App crashes when navigating from **Approved** -> **In Discussion**

godcr-proposals

### LINKED ISSUE #694: no space between icon and menu title
On the more page there is currently no space between the icon and the menu title

Screenshot from 2021-11-16 16-43-47

### PR DESCRIPTION
**Pull request by Sirmorrison**
Fix #598 
Fix #600
Fix #683
Fix #694 
- Add governance splace screen
- Set default label colour.
- Set font weight from Bold to semibold based on mockup
- Fix html renderer indentation bug.
- Add auto sync config setting feature
- Add governance setting feature
- Add space between icon and title label on settings page

### CODE REVIEW

File: ui/load/load.go [hunk 1]
```diff
@@ -213,6 +215,8 @@ func loadIcons() Icons {
 		ListGridIcon:             decredmaterial.NewImage(decredIcons["list_grid"]),
 		DecredSymbolIcon:         decredmaterial.NewImage(decredIcons["decred_symbol"]),
 		DecredSymbol2:            decredmaterial.NewImage(decredIcons["ic_decred02"]),
+		GovernanceActiveIcon:     decredmaterial.NewImage(decredIcons["governance_a"]),
```
Comment (dreacot) [inline ui/load/load.go]:
```suggestion
		GovernanceActiveIcon:     decredmaterial.NewImage(decredIcons["governance_active"]),
```
Why isn't this governance_active

also it seems two images were added `governance_a` and `governance_active`

**Review by dreacot — CHANGES_REQUESTED** [review]
1. This page shouldn't be on the governance page, but rather on the overview page, just the same way you have the recent transactions card with the see all button
Screenshot from 2021-11-16 16-47-01

2. after proposals are update there's a green checkmark that should show, it's currently not implemented
Screenshot from 2021-11-16 16-50-13

3. going to the governance page, and filtering for approved proposals, the scrollbar starts at the bottom of the screen
Screenshot from 2021-11-16 16-52-59

**Comment by dreacot** [comment]
The info icon as seen on the mockup doesn't seem to be implemented
Screenshot from 2021-11-16 17-06-00

**Comment by Sirmorrison** [comment]
>     1. This page shouldn't be on the governance page, but rather on the overview page, just the same way you have the recent transactions card with the see all button
>        Screenshot from 2021-11-16 16-47-01
> 
>     2. after proposals are update there's a green checkmark that should show, it's currently not implemented
>        Screenshot from 2021-11-16 16-50-13
> 
>     3. going to the governance page, and filtering for approved proposals, the scrollbar starts at the bottom of the screen
>        Screenshot from 2021-11-16 16-52-59

From the mockup, the overview page is the governance page.
image

and the page you proposals page holds the proposals filters.
image

**Review by dreacot — APPROVED** [review]

**Pull request merged**